.pre-commit-config.yaml

```diff
@@ -6,6 +6,13 @@ repos:
     exclude: README.md
   - id: check-yaml
   - id: check-json
+  - id: end-of-file-fixer
+    exclude: |
+      (?x)^(
+        .github/aiida/profile.yaml  |
```

Comment:
Ah, sorry. This file _needs_ to have a non-empty EOF line? 😮

Reply:
I don't think I'd add these files here at all, but I like adding the `end-of-file-fixer` hook though, so maybe just remove the `exclude` key and value?